dts/arm/st/stm32f303.dtsi

```diff
@@ -5,3 +5,11 @@
  */
 
 #include <st/stm32f3.dtsi>
+
+/ {
+
```

Comment:
nit, remove blank line

Reply:
which one, the one before or after the { ?

Reply:
The one after